src/app.rs

```diff
@@ -1,14 +1,15 @@
-use std::fs;
-
-use egui::{FontData, FontDefinitions, FontFamily, Visuals};
-
 use crate::{
     state::AppState,
-    widgets::{mainpanel::render_main_panel, sidebar::render_sidebar},
+    widgets::{add_account::AddAccount, mainpanel::render_main_panel, sidebar::render_sidebar},
 };
 
+use egui::{FontData, FontDefinitions, FontFamily, Visuals};
+
+use std::fs;
+
 pub struct App {
     state: AppState,
+    add_account_panel: AddAccount,
```

Comment:
this doesn't need to be persisted here, does it?

Reply:
Then it needs to be an `Option` and set to `None` if the panel is not shown

Reply:
I tested the constructor with a log and it seems like it really just got called once